AdobeStockAdminUi/Controller/Adminhtml/System/Config/TestConnection.php

```diff
@@ -12,14 +12,15 @@
 use Magento\AdobeImsApi\Api\ConfigInterface;
 use Magento\Backend\App\Action;
 use Magento\Backend\App\Action\Context;
+use Magento\Framework\App\Action\HttpPostActionInterface;
 use Magento\Framework\Controller\Result\Json;
 use Magento\Framework\Controller\Result\JsonFactory;
 use Magento\Framework\Controller\ResultInterface;
 
 /**
  * Controller used for testing connection to Adobe Stock API from stores configuration
  */
-class TestConnection extends Action
+class TestConnection extends Action implements HttpPostActionInterface
```

Comment:
This operation does not change the system - let's change it to Get and update connection.js as well